src/device/GPU_Device.cmake

```diff
@@ -15,6 +15,7 @@
 
 set( TEST_KERNEL_BINDIR ${PROJECT_BINARY_DIR}/test CACHE PATH
   "GPU kernel binary install location" )
+set( CUDA_DEVICE ON)
```

Comment:
What does this variable do?  Who needs access to it?

Ok - I see that it is used by `config.h`.  Maybe add a comment here to describe how it is used.

Reply:
I can add a comment. Should I cache this as variable as well?